examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/VectorUdfs.cs

```diff
@@ -56,39 +58,29 @@ public void Run(string[] args)
             spark.Stop();
         }
 
-        private static RecordBatch CountCharacters(
-            RecordBatch records,
+        private static FxDataFrame CountCharacters(
+            FxDataFrame dataFrame,
             string groupFieldName,
             string stringFieldName)
         {
-            int stringFieldIndex = records.Schema.GetFieldIndex(stringFieldName);
-            StringArray stringValues = records.Column(stringFieldIndex) as StringArray;
-
             int characterCount = 0;
 
-            for (int i = 0; i < stringValues.Length; ++i)
+            PrimitiveDataFrameColumn<int> characterCountColumn = new PrimitiveDataFrameColumn<int>(stringFieldName + "CharCount");
+            PrimitiveDataFrameColumn<int> ageColumn = new PrimitiveDataFrameColumn<int>(groupFieldName);
+            ArrowStringDataFrameColumn fieldColumn = dataFrame[stringFieldName] as ArrowStringDataFrameColumn;
```

Comment:
(nit) what does `fieldColumn` mean? Why that name?

Reply:
Hmm, I don't remember why I chose that anymore :). Fixed